src/azure-cli/azure/cli/command_modules/iot/custom.py

```diff
@@ -418,7 +425,9 @@ def iot_hub_create(cmd, client, hub_name, resource_group_name, location=None,
     storage_endpoint_dic['$default'] = StorageEndpointProperties(
         sas_ttl_as_iso8601=timedelta(hours=fileupload_sas_ttl),
         connection_string=fileupload_storage_connectionstring if fileupload_storage_connectionstring else '',
-        container_name=fileupload_storage_container_name if fileupload_storage_container_name else '')
+        container_name=fileupload_storage_container_name if fileupload_storage_container_name else '',
```

Comment:
should it be `fileupload_storage_container_name if fileupload_storage_container_name else None`? What's the difference between `None` and `""`?

Reply:
To clarify, would you rather them all be `None`, or just `fileupload_storage_container_name` ?
Was just trying to change as little as possible in existing code when adding a new parameter, it had existed as `''` before

Reply:
In reality I don't believe there's a difference in result on the service end, but I could be wrong.